Source/Stash.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+import Security
+
+public struct Stash {
+
+  enum Action {
+    case Insert, Fetch, Delete
+  }
+
+  // MARK: - Public methods
+
+  public static func password(service: String, account: String) -> String {
+    guard !service.isEmpty && !account.isEmpty else { return "" }
+
+    let query = [
+      kSecAttrAccount as String : account,
+      kSecAttrService as String : service,
+      kSecClass as String : kSecClassGenericPassword]
+
+    return Stash.query(.Fetch, query).1
+  }
+
+  public static func setPassword(password: String, service: String, account: String) -> Bool {
```

Comment:
@RamonGilabert what value would you like it to return if it should be optional? For me, `Bool` makes sense.